packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -434,9 +442,17 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
     function initialize() external {
         // SAFETY: Any revert in this function will bubble up to the DisputeGameFactory and
         // prevent the game from being created.
+        //
         // Implicit assumptions:
         // - The `gameStatus` state variable defaults to 0, which is `GameStatus.IN_PROGRESS`
 
+        // The VMStatus must indicate (1) 'invalid', to argue that disputed thing is invalid.
+        // Games that agree with the existing outcome are not allowed.
+        // NOTE(clabby): This assumption will change in Alpha Chad.
+        if (uint8(Claim.unwrap(rootClaim())[0]) != VMStatus.unwrap(VMStatuses.INVALID)) {
```

Comment:
I'm thinking about the case where an output is proposed, but no one can validate it because the block transition results in a panic (`VMStatuses.Panic`). Don't we want to prevent such outputs? Safety vs Liveness tradeoff.

Reply:
Good point. For now, we'll probably want games with `INVALID` or `PANIC` to be created. In alpha chad, we'll be able to change this to just be `VALID`.

Reply:
how would it change with Alpha Chad?

Reply:
Alpha chad will change the purpose of the game to proving an output root correct rather than incorrect - in addition we'll have output bisection, so the root claim won't have a status byte (since it'll be the output root).

The first state hash claim (if it's by an honest actor, which we can guarantee by deliberately setting the split level), should have the VM status as valid always in that world. Not set in stone, though.